msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -7,49 +7,79 @@ ms.localizationpriority: high
 
 # Create deep links
 
-You can create links to information and features within Teams. The scenarios where creating deep links are useful are as follows:
+Deep links are a navigation mechanism that you can use to connect users with information and features within Teams and Teams apps. Some scenarios where creating deep links can be useful are as follows:
 
 * Navigating the user to the content within one of your app's tabs. For instance, your app can have a bot that sends messages notifying the user of an important activity. When the user taps on the notification, the deep link navigates to the tab so that the user can view more details about the activity.
-* Your app automates or simplifies certain user tasks, such as creating a chat or scheduling a meeting, by pre populating the deep links with required parameters. This avoids the need for users to manually enter information.
+* Your app automates or simplifies certain user tasks, such as creating a chat or scheduling a meeting, by pre-populating the deep links with required parameters. This avoids the need for users to manually enter information.
```

Comment:
I was mostly talking about it as an improved method for navigation. I've tried to clarify it in the latest commits as a recommended method where possible, since some scenarios need deep links.